src/components/shared/Drawer.js

```diff
@@ -124,7 +126,7 @@ class DrawerComponent extends Component {
         </List>
         <Divider />
         <List>
-          <ListItem button>
+          <ListItem button onClick={this.props.onAboutClick}>
```

Comment:
no - there should be one method for each click handler - like onItemClick. And then in each on click you call it with different parameter like this:


```
<ListItem button onClick={() => this.props.onItemClick('about')}>

<ListItem button onClick={() => this.props.onItemClick('albums')}>

...

```